app/controllers/shared/api.php

```diff
@@ -110,6 +110,61 @@
 
 }, ['utopia', 'request', 'response', 'project', 'user', 'register', 'events', 'audits', 'usage', 'deletes'], 'api');
 
+
+App::init(function ($utopia, $request, $response, $project, $user) {
+    /** @var Utopia\App $utopia */
+    /** @var Utopia\Swoole\Request $request */
+    /** @var Appwrite\Utopia\Response $response */
+    /** @var Appwrite\Database\Document $project */
+    /** @var Appwrite\Database\Document $user */
+    /** @var Utopia\Registry\Registry $register */
+    /** @var Appwrite\Event\Event $events */
+    /** @var Appwrite\Event\Event $audits */
+    /** @var Appwrite\Event\Event $usage */
+    /** @var Appwrite\Event\Event $deletes */
+    /** @var Appwrite\Event\Event $functions */
+
+    $route = $utopia->match($request);
+
+    $isPreviliggedUser = Auth::isPreviliggedUser(Authorization::$roles);
```

Comment:
Can we fix the spelling of this method in this PR or save for another?
https://github.com/appwrite/appwrite/blob/7acfa98221c1e2a6a96731aabd7254ee2b2ae6c0/src/Appwrite/Auth/Auth.php#L217
Should be isPrivilegedUser

Reply:
I'll fix it in this one 👍🏻